zebra-script/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "zebra-script"
-version = "1.0.0-alpha.18"
+version = "1.0.0-alpha.19"
```

Comment:
Just asking to be sure. Is this expected to be larger than `18`?

Reply:
Yes, because we had a sprint where we only changed the script version, and we've never equalized the versions again.

(This problem will go away when we switch to beta.)